pkgs/applications/science/electronics/dataexplorer/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchurl
+, jdk
+, jre
+, ant
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dataexplorer";
+  version = "3.6.2";
+
+  src = fetchurl {
+    url = "mirror://savannah/dataexplorer/dataexplorer-${version}-src.tar.gz";
+    sha256 = "sha256-2e8qeoJh7z/RIowMtAd8PGcMPck5H8iHqel6bW7EQ0E=";
+  };
+
+  nativeBuildInputs = [ jdk ant makeWrapper ];
+
+  buildPhase = ''
+    ant -f build/build.xml dist
+  '';
+
+  doCheck = false;
+  # Missing dependencies (e.g. junit). Does not work.
+  #checkPhase = ''
+  #  ant -f build/build.xml check
+  #'';
+
+  installPhase = ''
+    ant -Dprefix=$out/share/ -f build/build.xml install
+
+    # see $installdir/DataExplorer for the upstream wrapper script but we make
```

Comment:
That does not explain why we want to do that.

Reply:
I tried to clarify the rationale. Also the [AUR pkg](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dataexplorer#n57) doesn't use the original launchers/wrappers.